### DISCUSSION TITLE
Plotly Card keeps shrinking in real time!!!

### DISCUSSION THREAD

**Discussion by luissimoes** [dbuezas/lovelace-plotly-graph-card#363] [Q&A] [answered]
this is very very strange.... let me try to explain.
- I have a dashboard in Grid layout Format. (code below); it works perfectly with ApexCharts (but I want to use Plotly instead)
- **as soon as I have 2 Plotly charts in one row of the Grid, that row starts to shrink **dynamically**** at a rate of the full height of the graph every 7 or 8 seconds

example:
I have one card in zone "detail1" and and another not in any zone: it works
if I have one card in "detail1" and another in "detail2" then the striking happens.

Any help please!.... this is key for me to use the cards....
Note: in single column this doesn't happen

- this happens in Mac and iPad.... as long as there are at least 2 columns

How this looks like (2 sequences.... of many in-between steps)
<img width="798" alt="Screenshot 2024-01-24 at 10 59 51" src="https://github.com/dbuezas/lovelace-plotly-graph-card/assets/42578174/96198b20-6d5c-4aca-a975-6343b3bc379d">
<img width="798" alt="Screenshot 2024-01-24 at 11 00 06" src="https://github.com/dbuezas/lovelace-plotly-graph-card/assets/42578174/80046fcb-a55a-47e3-9a84-a34b334a2fdc">

Here is the code for the grid

> grid-template-columns: repeat(auto-fit, minmax(200px, 30%))
grid-template-rows: auto
grid-template-areas: |
  "header header header"
  "view1 view1 view3"
  "detail1 detail2 detail3"
  "detail4 detail4 detail6"
  "group1 group2 group3"

thanks!!!

**Comment by dbuezas** [comment, marked as answer]
As a temporary workaround, try fixing the height:
```yaml
layout:
  height: 500
```

**Reply by luissimoes** [reply]
that works.... thanks!
workaround yes, but its ok and I can continue using it.

It's a strange thing though right?.... seems that the grid keeps communicating with the card?... not only at start of display?

thanks for your support!

**Reply by dbuezas** [reply]
Yeah, that's exactly what is happening. I don't remember the details, but some layouts (e.g mansonery) grow with the card while others start as big as the screen (e.g side bar), so this card tries to work best on both and that seems to have created this issue with the grid layout.
Good there's a workaround : )
